plugins/search-backend-module-pg/README.md

```diff
@@ -14,3 +14,31 @@ other plugins.
 
 See [Backstage documentation](https://backstage.io/docs/features/search/search-engines#postgres)
 for details on how to setup Postgres based search for your Backstage instance.
+
+## Optional Configuration
```

Comment:
This is great stuff!  Thanks for documenting.  Perhaps it'd be better in `/docs/features/search/search-engines.md`, though?  I don't know how often folks reference the README, and there's a link to the live page just above anyway!

Also: on that page, there's a reference to `PgSearchEngine.from()` that should be updated.

Reply:
I've added the same details to this page and updated to reference the `PgSearchEngine.fromConfig()`. I tend to look at both places, the documentation on Backstage.io doesn't always have the details, but it's always improving